test/clt-tests/comparison-overhead-json-sql/for_fuzzy_search.php

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/php
```

Comment:
I'm still not sure that placing "for_fuzzy_search" in the "overhead-json-sql" folder is a good idea. Also, I'm not sure that we really need to have a lot of different scripts to generate mock data. I think it's much better and worth it to use a single generic script that can do all the jobs we need, because maintenance of multiple scripts that are actually doing the same thing is much harder than maintaining a single one with a clear purpose.

Reply:
Creating a scripts folder and moving all scripts there, deleting unnecessary scripts